### PR TITLE
fix: `clientId` assignment for sandbox test

### PR DESCRIPTION
**Pull request by ttypic**
Updated `clientId` assignment for sandbox test

## Release Notes

- **New Features**
	- Enhanced chat client initialization by incorporating a `clientId`, improving user-specific room management.
	- Updated message handling logic for better error reporting and state management.

- **Bug Fixes**
	- Improved validation in message subscription methods to prevent errors from empty messages and invalid parameters.

- **Documentation**
	- Clarified method signatures and parameters for better understanding and usage.

These updates aim to provide a more robust and user-friendly chat experience.

### CODE REVIEW

File: chat-android/src/main/java/com/ably/chat/Messages.kt [hunk 1]
```diff
@@ -243,7 +243,7 @@ internal class DefaultMessages(
 
     init {
         channelStateListener = ChannelStateListener {
-            if (!it.resumed) updateChannelSerialsAfterDiscontinuity()
+            if (it.current == ChannelState.attached && !it.resumed) updateChannelSerialsAfterDiscontinuity()
         }
         channel.on(channelStateListener)
```
[the remark this answers is not in the document]
Comment (sacOO7) [reply chat-android/src/main/java/com/ably/chat/Messages.kt:248]: In this case, no need for explicit `it.current == ChannelState.attached` check internally

[the remark this answers is not in the document]
Comment (sacOO7) [reply chat-android/src/main/java/com/ably/chat/Messages.kt:248]: `channelStateListener` can be renamed to `attachListener`

[the remark this answers is not in the document]
Comment (ttypic) [reply chat-android/src/main/java/com/ably/chat/Messages.kt:248]: No, unfortunately it won't work, ably-java implementation is a bit odd, we can discuss this on call

[the remark this answers is not in the document]
Comment (sacOO7) [reply chat-android/src/main/java/com/ably/chat/Messages.kt:248]: Thanks for the explanation.

**Pull request merged**